mcsema/CFG/CFG.proto

```diff
@@ -129,18 +129,24 @@ message Variable {
   required  string      name = 2;
 }
 
-message StackVar {
-    required string     name = 1;
-    required int64      sp_offset = 2;
-    required int64      size = 3;
-    required bool       has_frame = 4;
-    required string     regname = 5;
+message Reference {
+  required  uint64   inst_ea = 1;
+  required  int64    offset = 2;
+}
+
+message StackVariable {
+  required string     name = 1;
+  required uint64     size = 2;
+  required int64      sp_offset = 3;
+  optional bool       has_frame = 4;
+  optional string     regname = 5;
+  repeated Reference  ref_eas = 6;
 }
 
 message GlobalVar {
```

Comment:
Is the `GlobalVar` message type being used? Can you rename it to `GlobalVariable` to be consistent with the rest of the variable-related message class names?